bitcoin/script/miniscript.cpp

```diff
@@ -326,6 +326,8 @@ bool DecomposeScript(const CScript& script, std::vector<std::pair<opcodetype, st
             // Decompose OP_EQUALVERIFY into OP_EQUAL OP_VERIFY
             out.emplace_back(OP_EQUAL, std::vector<unsigned char>());
             opcode = OP_VERIFY;
+        } else if (opcode >= OP_PUSHDATA1 && opcode <= OP_PUSHDATA4) {
```

Comment:
I think this check must also be performed on normal pushes (non-OP_PUSHDATAx).

Reply:
Arf, i messed up my patch (copy pasting patches from `bitcoin/` to `miniscript/`). It should use the `IsPushOpCode` introduced in this commit. Fixing now